api/src/pcapi/routes/adage/v1/serialization/prebooking.py

```diff
@@ -205,3 +216,7 @@ def _get_educational_offer_accessibility(offer: offers_models.Offer) -> str:
         disability_compliance.append("Visuel")
 
     return ", ".join(disability_compliance) or "Non accessible"
+
+
+def get_bookings_for_educational_year(educational_year_id: str) -> list[EducationalBooking]:
+    return EducationalBooking.query.filter(EducationalBooking.educationalYearId == educational_year_id)
```

Comment:
Je suis pas sûr que ce soit le bon filtre, plutôt:
```suggestion
    return EducationalBooking.query.join(EducationalYear).filter(EducationalYear.adageId == educational_year_id)
```

Reply:
pourquoi ? ce sont les mêmes valeurs (du moins sur testing)